static/app/views/performance/newTraceDetails/traceDrawer/details/transaction.tsx

```diff
@@ -409,13 +448,14 @@ export function TransactionNodeDetails({
         </tbody>
       </TraceDrawerComponents.Table>
       {project ? <EventEvidence event={event} project={project} /> : null}
+      <ReplaySection event={event} organization={organization} />
       {event.projectSlug ? (
         <Entries
           definedEvent={event}
           projectSlug={event.projectSlug}
           group={undefined}
           organization={organization}
-          isShare={false}
+          isShare
```

Comment:
Why are we changing this?

Reply:
This prop set to true prevents the replay section from the Entries component from rendering. We have our own replay section now.